app/components/DropDown/DropDownStyles.ts

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native';
+import {
+  Colors,
+  horizontalScale,
+  moderateScale,
+  verticalScale,
+} from '../../theme';
+
+export const styles = StyleSheet.create({
+  dropDownContainer: {
+    position: 'absolute',
+    right: 0,
+    backgroundColor: Colors.lightGreen,
+    borderRadius: 20,
+    borderWidth: moderateScale(1),
+  },
+  dropDownButton: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    backgroundColor: Colors.DarkBlue,
+    width: horizontalScale(145),
+    paddingVertical: verticalScale(4),
+    paddingHorizontal: horizontalScale(15),
+    borderRadius: 15,
```

Comment:
used theme scale functions

Reply:
We are instructed that when using border radius don't use any scale functions